### DISCUSSION TITLE
Is there a way to get a random record or order randomly with Ormar ORM?

### DISCUSSION THREAD

**Discussion by bolinocroustibat** [collerek/ormar#591] [Q&A]
Is there a way to get a random record or order randomly?

i.e. is there an equivalent to Django `.order_by('?')` or to SQLalchemy `.order_by(func.rand())`?

**Comment by bolinocroustibat** [comment]
I created an issue about that: https://github.com/collerek/ormar/issues/605

**Comment by collerek** [comment]
It's not yet implemented, so it's not really a bug. Will check what i can do about it.

**Reply by bolinocroustibat** [reply]
Thanks. Don't hesitate to change the label of the issue.
